config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -391,3 +391,46 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-1903-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs SIG-Windows release tests on K8s clusters provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+- interval: 10h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-1909
+  labels:
+    preset-service-account: "true"
+    preset-azure-aksengine: "true"
```

Comment:
We've re-named `preset-azure-aksengine` to `preset-azure-windows` in https://github.com/kubernetes/test-infra/pull/15774. `preset-azure-cred` should also be added for credential-related stuff.